vmdb/app/views/layouts/quadicon/_ext_management_system.html.haml

```diff
@@ -1,4 +1,12 @@
-- if @settings[:quadicons][@layout == 'ems_infra' ? :ems : :ems_cloud]
+- db = case @layout
+- when "ems_infra"
+  - :ems
+- when "ems_cloud"
+  - :ems_cloud
+- else
+  - :ems_container
```

Comment:
@Fryguy HAML is indent based and auto closes blocks based on indents . . . it's really fun when you want to comment sections out or add a nesting level.